roles/vdm/tasks/start_stop.yaml

```diff
@@ -0,0 +1,42 @@
+- name: start_stop - validate cadence version
+  ansible.builtin.fail:
+    msg: >
+      The schedule-start-stop.yaml transformer is not supported is not supported for cadences 2021.1 and older
```

Comment:
minor change request: Message needs update `is not supported` was repeated.

Reply:
Good catch, fixed.